ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -398,6 +398,31 @@ To retain the previous behaviour, set the value to `false`:
 +shutdown_interfaces_towards_undeployed_peers: false
 ```
 
+### Default MTU value changed from `9214` to `9194` for Dps1 and WAN HA interfaces
```

Comment:
Please also include the uplink interfaces for WAN platforms here. And the revert would be to use custom_platform_settings (include example)

Reply:
added